anoncreds/protocol/primary/primary_proof_builder.py

```diff
@@ -106,12 +106,12 @@ def __init__(self, wallet: ProverWallet):
                                      unrevealedAttrs.keys(), revealedAttrs)
 
     async def _initGeProof(self, schemaKey, eqProof: PrimaryEqualInitProof,
-                           c1: PrimaryClaim, predicate: Predicate) \
+                           c1: PrimaryClaim, predicate: Predicate, claim: Dict[str, AttributeValues]) \
```

Comment:
We use `attrs` in some cases, and `claims` in others. Please us the same name.
I think we can use the term ClaimAttributes. It means that the wallet method should be renamed to `submitClaimAttributes`